src/sign_msg_auth_token.c

```diff
@@ -40,14 +36,11 @@ static uint8_t set_result_auth_token(void) {
 
 #if defined(TARGET_STAX)
 
-static void start_review(void);
-static void ui_sign_message_auth_token_nbgl(void);
-static void rejectUseCaseChoice(void);
 static nbgl_layoutTagValueList_t layout;
```

Comment:
These variable names are still mixed(snake_case + camelCase).

Reply:
`nbgl_layoutTagValueList_t` is a struct that comes from the SDK so I won't be able to modify it.